tests/bson/bson-fromJSON_error-001.phpt

```diff
@@ -0,0 +1,23 @@
+--TEST--
+BSON\fromJSON(): invalid JSON
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"?>
+--FILE--
+<?php
+
+require_once __DIR__ . "/../utils/basic.inc";
+
+echo throws(function() {
+    fromJSON('foo');
+}, 'MongoDB\Driver\Exception\UnexpectedValueException'), "\n";
+
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECT--
+OK: Got MongoDB\Driver\Exception\UnexpectedValueException
+lexical error: invalid string in json text.
+                                       foo
+                     (right here) ------^
```

Comment:
Unfortunately, yes. It comes directly from yajl. We can attempt to shorten this to a one-liner at a later point.